packages/esm-appointments-app/src/config-schema.ts

```diff
@@ -1,5 +1,18 @@
 import { Type, validators } from '@openmrs/esm-framework';
 
+export const appointmentColumnTypes = [
+  'patientName',
+  'identifier',
+  'location',
+  'serviceType',
+  'status',
+  'date-time',
+  'provider',
+  'actions',
+] as const;
```

Comment:
Add commented t-function calls so these end up as translation keys in `en.json`.

e.g.

```
// t('patientName', 'Patient name')
```

Also fix `date-time` to `dateTime`